holoviews/plotting/bokeh/callbacks.py

```diff
@@ -611,9 +622,10 @@ def initialize(self, plot_id=None):
         }
         """],
         css_classes=["popup-close-btn"])
+        self._popup_position = stream.popup_position
         self._panel = Panel(
             position=XY(x=np.nan, y=np.nan),
-            anchor="top_left",
+            anchor=stream.popup_anchor or POPUP_POSITION_ANCHOR.get(self._popup_position, 'top_left'),
```

Comment:
```suggestion
            anchor=stream.popup_anchor or POPUP_POSITION_ANCHOR[self._popup_position],
```

I think something has gone wrong if `self._popup_position` is not in `POPUP_POSITION_ANCHOR`.